docs/variables.rst

```diff
@@ -56,3 +56,141 @@ This bool Series identifies Pensioner employment status.
 Alleinerziehend
 ---------------
 This bool Series identifies single parents.
+
+
+
+.. _wohnort_ost:
+
+Wohnort
+-------
+This BoolSeries identifies whether location is in former East or West Germany.
+
+
+
+.. _geringfügig_beschäftigt:
+
+Geringfügig beschäftigt
+-----------------------
+This bool Series identifies marginal employment.
+
+
+.. _in_gleitzone:
```

Comment:
`_in_gleitzone`, `pflegev_beitr_m`, `ges_krankenv_beitr_m`, `arbeitsl_v_beitr_m` shoulnd't be in the list of basic data variables and thus not be explained here. In the you referenced them right with :func: and not :ref: !